pkg/ccl/streamingccl/streamclient/client.go

```diff
@@ -118,7 +118,7 @@ type Client interface {
 type LogicalReplicationClient interface {
 	Client
 
-	PartitionSpans([]roachpb.Span) (Topology, error)
+	PlanLogicalReplication([]roachpb.Span) (Topology, error)
```

Comment:
Heh, it doesn't matter because nothing is using this interface.  But this signature is wrong.

Reply:
Done.